coordinator/src/routes.rs

```diff
@@ -160,7 +161,11 @@ pub fn router(
         .route("/api/orderbook/websocket", get(websocket_handler))
         .route("/api/trade", post(post_trade))
         .route("/api/rollover/:dlc_channel_id", post(rollover))
+        // Deprecated: we just keep it for backwards compatbility as otherwise old apps won't
+        // pass registration
         .route("/api/register", post(post_register))
+        .route("/api/users", post(post_register))
+        .route("/api/users/:trader_pubkey", get(get_user))
```

Comment:
🔧 That api is concerning for me. If somebody learns about a users node id they can use this api to query user data.

I propose we either validate a users signature on that api or rather put that information into the authenticated message received by the app after a successful login. (there we already verify the users signature)